app/scripts/modules/core/src/managed/toggleResourceManagement.ts

```diff
@@ -43,34 +63,52 @@ export const toggleResourcePause = (
   });
 };
 
+const getInterstitialBodyText = (resourceSummary: IManagedResourceSummary) => {
+  const { status } = resourceSummary;
+  let body = `
+        <p>🌈 <b>Spinnaker is continuously managing this resource.</b></p>
+        <p>If you need to temporarily stop Spinnaker from managing this resource — for example, if something is wrong
+        and manual intervention is required — you can pause management and resume it later.</p>`;
+  body += viewConfigurationByStatus[status]?.pauseWarning ?? '';
+  body += multiRegionWarning(resourceSummary);
+  return body;
+};
+
 const getPopoverToggleBodyText = (resourceSummary: IManagedResourceSummary) => {
-  const { isPaused, locations, status } = resourceSummary;
-  const regions = locations.regions.map(r => r.name).sort();
+  const { isPaused, status } = resourceSummary;
   let body = '';
   if (!isPaused) {
     body += `
         <p>
           While a resource is paused, Spinnaker will not take action to resolve drift from the declarative configuration.
         </p>`;
-    body += viewConfigurationByStatus[status]?.popoverPauseWarning ?? '';
+    body += viewConfigurationByStatus[status]?.pauseWarning ?? '';
   } else {
     body += `
         <p>
           Spinnaker will resume taking action to resolve drift from the declarative configuration.
         </p>
       `;
   }
-  if (regions.length > 1) {
-    body += `
-        <p>
-          <div class="horizontal top sp-padding-m alert alert-warning">
-            <i class="fa fa-exclamation-triangle sp-margin-m-right sp-margin-xs-top"></i>
-            <span>${
-              isPaused ? 'Resuming' : 'Pausing'
-            } management of this resource will affect the following regions: <b>${regions.join(', ')}</b>.
-            </span>
-          </div>
-        </p>`;
-  }
+  body += multiRegionWarning(resourceSummary);
   return body;
 };
+
+const multiRegionWarning = (resourceSummary: IManagedResourceSummary) => {
+  const { isPaused, locations } = resourceSummary;
+  const regions = locations.regions.map(r => r.name).sort();
+  if (regions.length < 2) {
+    return '';
+  }
+  return `
```

Comment:
is this inline html temporary while we wait for a react confirm thinggy ?

Reply:
yah, is temp